src/raw.rs

```diff
@@ -501,8 +502,10 @@ pub fn load_string_buffer(rdb: *mut RedisModuleIO) -> Result<RedisBuffer, Error>
 
 #[allow(clippy::not_unsafe_ptr_arg_deref)]
 pub fn replicate(ctx: *mut RedisModuleCtx, command: &str, args: &[&str]) -> Status {
```

Comment:
Yes it could and it probably should, but the changes on this PR is getting to big so I wanted to avoid it. We should make a separate PR to arrange all those pointers use.